dev_guide/compute_resources.adoc

```diff
@@ -303,15 +324,31 @@ attempting to recover memory.
 more. In the event of an out of memory event, it will only be killed if there
 are no more *BestEffort* or *Burstable* containers on the system.
 
+With Ephemeral Storage Resources _(if your administrator has enabled the ephemeral-storage tech preview in {product-title} 3.10)_: ::
```

Comment:
ephemeral storage is not tied to qos, and pods getting evicted are independent of qos.

see: https://github.com/kubernetes/kubernetes/blob/master/pkg/kubelet/eviction/eviction_manager.go#L367

the kubelet will evict any pod that exceeds its limit with no ranking order.

Reply:
to clarify, pod eviction associated with ephemeral storage is not tied to qos (where it is w/ memory).

Reply:
Missed this in cleanup.  Removing the section on ephemeral storage here.